website/docs/addon_3dequalizer_admin.md

```diff
@@ -12,3 +12,17 @@ import versions from '@site/docs/assets/json/Ayon_addons_version.json'
 <ReactMarkdown>
 {versions.Equalizer_Badge}
 </ReactMarkdown>
+
+## Overview
+
+To get 3DEqualizer (3DE) addon running, we need to instal Qt Python bindings - for that we are using pre-launch hook that is automatically installing PySide2/6 into (3DE) Python. You can use ```TDE4_HOME``` environment variable to set the location, or it will be detected from the AYON settings.
```

Comment:
```suggestion
To get 3DEqualizer (3DE) addon running, we need to install Qt Python bindings - for that we are using pre-launch hook that is automatically installing PySide2/6 into (3DE) Python. You can use ```TDE4_HOME``` environment variable to set the location, or it will be detected from the AYON settings.
```